docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -0,0 +1,37 @@
+= What's New in Hazelcast Platform Operator
+:description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.0.
+
+{description}
+
+NOTE: For new and improved features in Hazelcast Platform 5.5, see xref:{page-latest-supported-hazelcast}@hazelcast:ROOT:whats-new.adoc[What's new in Hazelcast Platform 5.5].
+
+== Community Edition and Repository Changes
+
+In March 2024, we made some updates to the Community Edition of the Hazelcast Platform, including making the Platform Operator 5.13 closed-source and its cluster creation capabilities will no longer be available in the Community Edition. 
+
+You can find more information here: https://hazelcast.com/blog/changes-to-community-edition/
+
+
+== WAN Replication Performance and Usability Enhancements
+
+For performant WAN Replication, all members should be connected directly by source clusters. To achieve this topology, we have intorudced `WithExposeExternally` parameter for Advanced WAN Network configuration at Hazelcast custom resource, see xref:advanced-networking.adoc#configuring-wan-protocol-sockets[Configuring WAN Protocol Sockets]
```

Comment:
```suggestion
For best performance using WAN Replication, all members should be connected directly by source clusters. To achieve this topology, we have introduced the `WithExposeExternally` parameter for Advanced WAN Network configuration. For more info, see xref:advanced-networking.adoc#configuring-wan-protocol-sockets[Configuring WAN Protocol Sockets].
```

Not sure what ' at Hazelcast custom resource' means here so suggestion includes taking it out...

Reply:
`Custom Resource` is kubernetes term, see https://kubernetes.io/docs/concepts/extend-kubernetes/operator/

We should keep it.

Reply:
I have addressed other parts with https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/228/commits/39051d12189c8536137a6a807913ea8e399f2dc2

Reply:
It's maybe just the preposition that reads wrong to me i.e. "at". 

...we have introduced the `WithExposeExternally` parameter for configuring the Advanced WAN Network for the Hazelcast custom resource.